GoogleForCreators/web-stories-wp
# Fix Video Captions behaviour

## Summary


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
- Add a video
- See the placement of the caption section in the design panel.
- Upload a caption.
- Set the video as background.
- Verify the placement (order) in the design panel is the same for the caption section.
- Verify that the caption for the background video is still set.


---



Fixes #4899
